lib/utils/load.rb

```diff
@@ -193,9 +198,9 @@ def load_environment(wanted_envs)
       load_environment(inh_env)
     end
   end
-  log_info("Loading environment '#{wanted_envs}'")
+  log_info("Loading environment '#{wanted_envs}'") if setup_commands
```

Comment:
Is this needed? We are still loading the environment vars, so why remove the log message?

Reply:
I think I have corrected it with a better log message